examples/failing/ProgrammableTypeErrorsTypeString.purs

```diff
@@ -0,0 +1,18 @@
+-- @shouldFailWith NoInstanceFound
+
+module Main where
+
+import Prelude
+import Control.Monad.Eff (Eff)
+import Control.Monad.Eff.Console (log)
+
+newtype MyType a = MyType a
+
+instance cannotShowFunctions :: Fail ("Don't want to show" <> TypeString (MyType a) <> "because.") => Show (MyType a) where
```

Comment:
You know what might be neat later - type level string interpolation:

```purescript
instance cannotShowFunctions :: Fail "Don't want to show {{MyType a}} because." => Show (MyType a)
```